content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,175 @@
+        $ kubectl apply -f deployment-injected.yaml
+        {{< /text >}}
+
+## Operational Details
+
+The Istio CNI plugin handles Kubernetes pod create and delete events and does the following:
+
+1.  Identify Istio user application pods with Istio sidecars requiring traffic redirection
+1.  Perform pod network namespace configuration to redirect traffic to/from the Istio sidecar
+
+### Identifying pods requiring traffic redirection
+
+The Istio CNI plugin identifies pods requiring traffic redirection to/from the
+accompanying Istio proxy sidecar by checking that the pod meets all of the following conditions:
+
+1.  The pod is NOT in a Kubernetes namespace in the configured `exclude_namespaces` list
+1.  The pod has a container named `istio-proxy`
+1.  The pod has more than 1 container
+1.  The pod has no annotation with key `sidecar.istio.io/inject` OR the value of the annotation is `true`
+
+### Traffic redirection details
+
+To redirect traffic in the application pod's network namespace to/from the Istio proxy sidecar, the Istio
+CNI plugin configures the namespace's iptables.  The following table describes the parameters to the
+redirect functionality.  The default values for the parameters are able to be overridden by setting the
+corresponding application pod annotation key.
+
+| Annotation Key | Values | Default | Description |
+|----------------|--------|---------|-------------|
+| `sidecar.istio.io/interceptionMode`| `REDIRECT`, `TPROXY` | `REDIRECT` | The iptables redirect mode to use. |
+| `traffic.sidecar.istio.io/includeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | "*" | Optional comma separated list of IP ranges in CIDR form to redirect to the sidecar proxy.  The default value of "*" redirects all traffic. |
+| `traffic.sidecar.istio.io/excludeOutboundIPRanges` | `<IPCidr1>,<IPCidr2>,...` | | Optional comma separated list of IP ranges in CIDR form to be excluded from redirection.  Only applies when `includeOutboundIPRanges` is "*". |
+| `traffic.sidecar.istio.io/includeInboundPorts` | `<port1>,<port2>,...` | Pod's list of `containerPorts` | Comma separated list of inbound ports for which traffic is to be redirected to the Istio proxy sidecar.  The value of "*" redirects all ports. |
+| `traffic.sidecar.istio.io/excludeInboundPorts` | `<port1>,<port2>,...` | | Comma separated list of inbound ports to be excluded from redirection to the Istio sidecar proxy.  Only valid when `includeInboundPorts` is "*". |
+
+### Logging
+
+The Istio CNI plugin is run by the container runtime process space, and, therefore, the log entries are added under
+the `kubelet` process.
+
+## Compatibility with other CNI plugins
+
+The Istio CNI plugin maintains compatibility with the same set of CNI plugins as the current `NET_ADMIN`
+`istio-init` container.
+
+The Istio CNI plugin operates as a chained CNI plugin which means its configuration is added to the existing
```

Comment:
RO.  "change which means" to ".  This means"

Reply:
done